addons/x_lib/functions/misc/fnc_wait.sqf

```diff
@@ -28,7 +28,7 @@ Author:
 ---------------------------------------------------------------------------- */
 
 private ["_condCode", "_maxDuration", "_traceComp", "_params", "_startTime", "_endTime", "_val"];
-_condCode = if (typeName(_this select 0) == "CODE") then {_this select 0} else {compile(_this select 0)};
+_condCode = if ((_this select 0) isEqualTo {}) then {_this select 0} else {compile(_this select 0)}; // TODO; remove compile
```

Comment:
Can't remove compile in this state. You could remove the `string` argument type from the header and then remove compile and make string a illegal argument. For that you have to check every usage of this function though.

Reply:
Reverted back to compile